### PR TITLE
[Assembly v1] CardContainer

### PR DESCRIPTION
**Pull request by katydecorah**
This PR merges into #481 and updates the CardContainer component to Assembly v1.

- [x] Run codemod and look out for additional changes
- [x] Check docs-prose.css for any component-specific classes that should be updated or removed, including changes to the color scale.
- [x] Visually check test cases app for component

Current | Proposed
---|---
screencapture-192-168-7-223-9966-CardContainer-2021-09-14-11_26_01 | screencapture-192-168-7-223-9966-CardContainer-2021-09-14-11_23_47

## QA checklist

- [x] No errors logged to console.
- [x] Accessibility score in [Chrome DevTools Audit/Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools) is 100% with Lighthouse version: `#.#.#`.
- [x] Component is accessible at mobile-size viewport.
- [x] Component is accessible at tablet-size viewport.
- [x] Component is accessible at laptop-size viewport.
- [x] Component is accessible at big-monitor-size viewport.

Open the test cases app locally on:

- [x] Chrome, no errors logged to console.
- [x] Firefox, no errors logged to console.
- [x] Safari, no errors logged to console.
- [x] Edge, no errors logged to console.
- [x] Mobile Safari, no errors logged to console.
- [x] Android Chrome, no errors logged to console.

## Before merge

- [x] Add entry to CHANGELOG.md to describe changes.
- [ ] If updating dependencies or creating a release, run `npx browserslist@latest --update-db` to update the [browser data](https://github.com/browserslist/browserslist#browsers-data-updating) and commit any changes to package-lock.json.

### CODE REVIEW

File: src/components/card-container/card-container.js [hunk 1]
```diff
@@ -4,45 +4,40 @@ import classnames from 'classnames';
...
-      'grid grid--gut36': !props.fullWidthCards
-    });
-    const renderedCards = props.cards.map((card, index) => {
-      return (
-        <div key={index} className={cardClasses}>
-          {card}
-        </div>
-      );
-    });
+    const { fullWidthCards, cards, title, path, cardColSize } = this.props;
```
Comment (katydecorah) [inline src/components/card-container/card-container.js:7]: 💡 Tidied up the code a bit.

File: src/components/card-container/card-container.js [hunk 2]
```diff
@@ -51,7 +46,7 @@ CardContainer.propTypes = {
   path: PropTypes.string,
   fullWidthCards: PropTypes.bool,
   cards: PropTypes.arrayOf(PropTypes.node).isRequired,
-  cardColSize: PropTypes.oneOf([1, 2, 3, 4, 5, 6])
+  cardColSize: PropTypes.oneOf(['1/4', '1/3', '1/2'])
```
Comment (katydecorah) [inline src/components/card-container/card-container.js:49]: What do you think about limited the `cardColSize` options? I don't think any of the (soon-to-be) deprecated sizes are in use.

Comment (malwoodsantoro) [reply src/components/card-container/card-container.js:49]: I think the limited options make sense! I do see cardColSize `5` used here https://github.com/mapbox/help/blob/publisher-production/src/pages/tutorials/create-a-map-with-data-visualization-component.md#choose-a-visualization-type, but it seems that we could change this to `1/2` 🤔 as well as https://github.com/mapbox/help/blob/publisher-production/src/pages/getting-started/add-markers.md#use-mapbox-gl-js

**Review by malwoodsantoro — APPROVED** [review]
🥇

File: src/components/card-container/card-container.js (hunk 2 above)
Comment (katydecorah) [reply src/components/card-container/card-container.js:49]: Thanks for catching that! I agree seems like `1/2` would be ideal there!

**Pull request merged**